pyproject.toml

```diff
@@ -120,6 +120,7 @@ ftfy = { version = "6.1.1", optional = true }
 regex = { version = "2023.8.8", optional = true }
 huggingface_hub = { version = "^0.17.3", optional = true }
 pymilvus = "2.3.1"
+pytest-asyncio = "^0.21.1"
```

Comment:
It should be part of the dev dependencies.

Reply:
Also, I am not sure if we really need this package btw.

Reply:
@deshraj thank you, I've added it to dev dependency. It's actually needed by a test for the Poe bot, which uses async code that's not currently running and is being skipped.